docs/data-guidelines.md

```diff
@@ -172,6 +174,24 @@ For Safari for iOS, use the iOS version number, not the Safari version number or
 
 This versioning scheme came at [Apple's request, in #2006](https://github.com/mdn/browser-compat-data/issues/2006#issuecomment-457277312).
 
+## Addition of browsers
+
+To add a new browser to BCD, we need evidence of (in decreasing order of importance):
+
+- a compelling downstream-consumer story (e.g., MDN or caniuse express an interest, or someone is planning to do something with the data that might plausibly grow BCD's reach)
+- reviewers (e.g., two or more people with interest and ability to test data relating to new and existing releases, or at least one reviewer acting on behalf of the vendor in BCD)
+- published release information (e.g., release notes with version numbers and dates, etc.)
+- documentation (e.g., how to get and test a feature in that browser, links to resources that might help with it, etc.)
+
```

Comment:
Can we add a trailing paragraph with something like "This guideline was proposed in #7238 and adopted in #7244." to both guidelines? This would align us with the template up on line 27.

Reply:
> Can we add a trailing paragraph with something like "This guideline was proposed in #7238 and adopted in #7244." to both guidelines?

Yup — pushed a commit here with that addition